ci-operator/step-registry/openshift/e2e/aws/single-node/graceful-shutdown/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- assisted-cicd
```

Comment:
Assisted people don't own single-node, please use the list of people from other single node tests

Reply:
for this task, I agree
do you want me to add a group for sno?

Reply:
>for this task, I agree
wdym? agree with what?

And yes, please open a group for sno

Reply:
agree to just replace with the list of people :)